src/Core/Domain/MerchandiseReturn/QueryResult/OrderDetailCustomization.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\MerchandiseReturn\QueryResult;
+
+use PrestaShop\PrestaShop\Adapter\Entity\Product;
+
+/** @todo at this point duplicate of PrestaShop\PrestaShop\Core\Domain\Order\QueryResult\OrderProductCustomizationForViewing */
```

Comment:
Mmmmm ... so I guess you want to know if this is a "good" duplicate" or a "bad" duplicate ?

Reply:
Yeah, I wonder if it's okay that it's duplicated and should I just leave it that way.

Reply:
I dont know yet. We are exploring our own business domain, and we are struggling with the following questions:
- a cart and an order share a lot of things, are they the same thing ?
- a cart item and an order detail share a lot of things, are they the same thing ?
- a product and an order detail share a lot of things, are they the same thing ?
- (and here is yours) an "order product" and an "order detail" share a lot of things, are they the same thing ?

Reply:
So let's duplicate them for now. if we find they are the same thing later we might merge them. But it is easier to deal with wrong duplicates than with wrong abstraction.